### PR TITLE
Blog app - controllers : Pull Request

### PR DESCRIPTION
**Pull request by belaymit**
## Description
**_In this project_**, I created all controllers and views for my **Blog app**. As I won't be processing any data yet, I added simple placeholders for my views with plain HTML.

## Project requirements

- Important: do NOT use ``rails generate scaffold`` command. The point of this project is to show you how to set up controllers by yourself.
- Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:

image

- For each URL you should create:
  - A route.
  - An action in the correct controller.
  - A view file.
  
- View files should include only placeholders written in plain HTML, e.g:
  - Here is a list of posts for a given user
- Make sure that you added code to handle only the 4 required URLs.

- Make sure that when you run the rails routes command in your console, you can see the following output:

image

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -2,4 +2,36 @@ class PostsController < ApplicationController
...
+
+  def destroy
+    # to do
+  end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
```
Comment (Gambit142) [inline app/controllers/posts_controller.rb]: - [ ] You used `rails generate scaffold` to create the `User`, `Post`, `Comment` and `Like` models. The essence of building the application from the scratch is to understand the MVC pattern/framework in Ruby on Rails and how they connect to the database. Kindly start this project without using the `rails generate scaffold`. 👍 😃

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @belaymit 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- README.md is professional ✔️ 
- Pull request title is professional ✔️ 
- Pull request summary is descriptive ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (belaymit) [reply app/controllers/posts_controller.rb]: Hello @Gambit142 , as per the project requirement, i.e. ``do NOT use rails generate scaffold command.``, I didn't use scaffold command, I do not know why did you say that.

**Comment by Gambit142** [comment]
Hi @belaymit , from the files created in the application especially those in the `app/view`, and those in `rspec` directory, I could see that you used a generator to create all models. 👍

**Review by Davidosky007 — APPROVED** [review]
Hi @belaymit 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Properly created controllers 👍 
* Well written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**